draft-scheffenegger-congress-rfc5033bis.md

```diff
@@ -292,6 +291,39 @@ trip times comparable bandwidth during backoff.
 alternate congestion control algorithm, the proposal should
 explore how bandwidth is shared among the competing flows.
 
+(*)
+: Utilising More than one Path.
+
+: An IETF transport is not tied to a specific Internet path.
+The set of routers forming the path can and do change with time,
+this will also cause the properties of the path to change with respect to time.
+Congestion control needs to be designed to take into account
+these changes in the path characteristics.
+Some transports permit more than one path to be differentiated and used by
+a single connection at the sender. These are called multipath transport protocols.  
+A multipath sender can schedule which packets travel on which of its active paths.
+This enables a tradeoff in timeliness and reliability.
+
+One use is to provide fail-over from one path to
+another when the original path is no longer viable or to switch the traffic from
+one path to another when this is expected to improve performance
+(latency, throughput, reliability, cost).
+A design independently needs to track the congestion state of each path,
+and needs to demonstrate independent congestion control for each path being used.
+
+A concurrent transport protocol simultaneously
+schedules multiple flows to aggregate the capacity of multiple paths.
+This introduces additional implications:
+The Internet provides no guarantee that different paths
+(e.g., using different endpoint addresses) are disjoint,
+therefore a design needs to evaluate the potential
+harm to other flows (including single path connections)
+that share a common bottleneck
+(or share resources that are coupled between different paths,
+such as an overall capacity limit).
+At the time of writing, there are no IETF standards for concurrent
+multipath usage in the general Internet.
+
```

Comment:
We do have a proposed standard for TCPM (RFC 8684). We also have active work on QUIC Multipath, which could very well end up approved before 5033bis. It might be better to say nothing.

Reply:
THAT is a good question... maybe @Martin  for more clue on this. RFC 8684 states: "it is necessary to couple the congestion windows in use on each subflow, in order to push most traffic to uncongested links", although RFC6356 is EXP

and then MP-DCCP (draft-ietf-tsvwg-multipath-dccp) was asked to say "If, and
   when, the IETF specifies a method for concurrent usage of multiple
   paths for the general Internet, the framework specified in this
   document could be used to provide an IETF recommended method for MP-
   DCCP."
... I'll let Martin comment first.

Reply:
Though resolved - at least enough to land something before we do the rewrite. This topic naturally will require some WG review.